app/models/book_indexing.rb

```diff
@@ -0,0 +1,72 @@
+class BookIndexing
```

Comment:
if we rename this, my preference would be something like BookIndexingJob or the like.  because it mostly accurately represents a job (in the domain world)...it contains state and a few other things (like timestamps).  

i do agree we should strive toward the best naming we can muster here...the word indexing is kind overloaded.

Reply:
@jpslav and I chatted more on this.   it was a miscommunication on my part that this class should be a state of indexing, and not a "activity log".  so...will rename to JP's recommendation.